### PR TITLE
Added "Find ESTP System"; Removed "Early Adopter"

### PR DESCRIPTION
**Pull request by julieplummer20**

### CODE REVIEW

File: tutorials/abap-s4hanacloud-procurement-purchasereq-checks/abap-s4hanacloud-procurement-purchasereq-checks.md [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
 - How to log on to SAP S/4HANA Cloud ABAP Environment
 - How to create an ABAP package
 - How to find relevant existing `BAdI` enhancement spots for your line of business, in this case **Materials-Management-Purchasing**
-- How to implement checks for the field quantity and delivery date during creation of purchase requisition item
+- How to implement a `BAdI` that checks for the field quantity and delivery date during creation of purchase requisition item
+
+
+
+## Intro
+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?locale=en-US).
```
Comment (MichaelCzcz) [inline tutorials/abap-s4hanacloud-procurement-purchasereq-checks/abap-s4hanacloud-procurement-purchasereq-checks.md:31]: Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21

File: tutorials/abap-s4hanacloud-procurement-purchasereq-fldctrl/abap-s4hanacloud-procurement-purchasereq-fldctrl.md [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
+## You will learn
 - How to logon to SAP S/4HANA Cloud ABAP Environment
 - How to create an ABAP package
 - How to find relevant existing `BAdI` enhancement spots for your line of business, in this case **Materials-Management-Purchasing**
 - How to hide a field, **Price**, by changing the status of a field control
 
-### Overview: Defining and implementing the enhancement
+
+## Intro
+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?locale=en-US).
```
Comment (MichaelCzcz) [inline tutorials/abap-s4hanacloud-procurement-purchasereq-fldctrl/abap-s4hanacloud-procurement-purchasereq-fldctrl.md:28]: Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21

File: tutorials/abap-s4hcloud-procurement-po-customlogic-tracing/abap-s4hcloud-procurement-po-customlogic-tracing.md [hunk 3]
```diff
@@ -1,86 +1,89 @@
...
-## Details
-### You will learn
+## You will learn
 - How to create custom logic for an existing custom field in the app **Custom Fields**
 - How to run a trace for the Fiori app **Create Purchase Order** in the app **Custom Logic Tracing**
 - How to check the output value of this custom field using **Custom Logic Tracing**
 - How to check the custom logic of this custom field, in **Custom Fields**
 
+## Intro
+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?locale=en-US).
```
Comment (MichaelCzcz) [inline tutorials/abap-s4hcloud-procurement-po-customlogic-tracing/abap-s4hcloud-procurement-po-customlogic-tracing.md:35]: Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
I think that if a note is the only text in the Intro section, it does not have to be formatted as a note.

**Comment by julieplummer20** [comment]
Hi,

“I think that if a note is the only text in the Intro section, it does not have to be formatted as a note.”

It isn’t 😉:

***@***.***

I think it should be separate because the rest of the intro is specific to this tutorial “Implement a BAdI”, whereas the Note is generic to all SAP ABAP Cloud Environment tutorials.

However, I have removed the word “NOTE”.

BR Julie.

File: tutorials/abap-s4hanacloud-procurement-purchasereq-fldctrl/abap-s4hanacloud-procurement-purchasereq-fldctrl.md [hunk 4]
```diff
@@ -251,6 +260,7 @@ Check that yours is the implementation that will be called:
 ### More Information
```
Comment (MichaelCzcz) [inline tutorials/abap-s4hanacloud-procurement-purchasereq-fldctrl/abap-s4hanacloud-procurement-purchasereq-fldctrl.md:260]: Please use sentence case.

File: tutorials/abap-s4hanacloud-procurement-purchasereq-checks/abap-s4hanacloud-procurement-purchasereq-checks.md [hunk 5]
```diff
@@ -288,7 +297,7 @@ Your new order appears in the Overview List.
 ### More Information
```
Comment (MichaelCzcz) [inline tutorials/abap-s4hanacloud-procurement-purchasereq-checks/abap-s4hanacloud-procurement-purchasereq-checks.md:297]: Please use sentence case.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/abap-environment-external-api/abap-environment-external-api.md [hunk 6]
```diff
@@ -8,156 +10,244 @@ author_name: Julie Plummer
...
+Run your class in the console (`F9`). The output should look something like this:
+&nbsp;
+<!-- border -->
+![step10a-output](step10a-output.png)
 
 
 ### Test yourself
 
 
+### More Information
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-external-api/abap-environment-external-api.md:244]: Sentence case please :-)

File: tutorials/abap-environment-external-api/abap-environment-external-api.md [hunk 7]
```diff
@@ -8,156 +10,244 @@ author_name: Julie Plummer
...
+      ENDMETHOD.
+
+    ```
+
+2. Check your syntax (`Ctrl+F2`).
+
+3. Format (`Ctrl+F1`), save (`Ctrl+S`) and activate (`Ctrl+F3`) your class.
+
+
+### Run class in ABAP Console
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-external-api/abap-environment-external-api.md:233]: Shouldn't it be "console" with a lower case C or is ABAP Console a product name?

File: tutorials/abap-environment-external-api-json/abap-environment-external-api-json.md [hunk 8]
```diff
@@ -0,0 +1,339 @@
...
+
+<!-- border -->
+![step4d-class-editor](step4d-class-editor.png)
+
+### Copy code
+
+
+To be able to access the external service, you must:
+
+- create an HTTP client that points to your communication scenario and service id
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-external-api-json/abap-environment-external-api-json.md:224]: Shouldn't this an the other line items start with upper case?

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please fix the image links! Border and link should be in one line

**Comment by julieplummer20** [comment]
Hi,

Discussed with Merve.

This is no longer as urgent, so I have sent only to QA anyway.

So next week is now fine.

“Shouldn't it be "console" with a lower case C or is ABAP Console a product name?”
It is Console (upper case) in the official docu. cf “SQL Console”.


“Shouldn't this an the other line items start with upper case?”

Not sure . Have changed to upper case.



Thanks for all your help.

Best wishes,

Julie.

**Pull request merged**